unitTests/ui/ResponsiveComponents/ResponsiveComponentsManagerTest.ts

```diff
@@ -40,6 +45,25 @@ export function ResponsiveComponentsManagerTest() {
       jasmine.clock().uninstall();
     });
 
+    it(`when registering a tab with #enableResponsiveMode false and a responsive facet,
+    when triggering the #afterInitialization event,
+    it calls the #register instance method to make the facet responsive`, () => {
+      const tabOptions: ITabOptions = { enableResponsiveMode: false };
+      const tab = Mock.optionsComponentSetup<Tab, ITabOptions>(Tab, tabOptions);
+
+      ResponsiveComponentsManager.register(ResponsiveTabs, root, Tab.ID, tab.cmp, tabOptions);
+
+      const facetOptions: IFacetOptions = {};
+      const facet = Mock.optionsComponentSetup<Facet, IFacetOptions>(Facet, facetOptions);
+
+      ResponsiveComponentsManager.register(ResponsiveFacets, root, Facet.ID, facet.cmp, facetOptions);
+
+      const spy = spyOn(responsiveComponentsManager, 'register');
+      root.trigger(InitializationEvents.afterInitialization);
+
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
```

Comment:
This test succeeds in isolation but fails when run as part of the entire suite. The reason is the `remainingComponentInitializations` counter is static and many other tests increment it.

I can make the counter property public to set it to zero, or remove the unit test. Let me know which you prefer

Reply:
Would resetting the counter using square brackets work? E.g., `ResponsiveComponentsManager['remainingComponentInitializations'] = 0`?

Reply:
hmm, that could be a good compromise. We will not have type-safety if the counter gets renamed, but the property stays private and we keep type-safety inside the source code. I will push this solution to fix the build.